### PR TITLE
Update variable names in team_test

### PR DESCRIPTION
**Pull request by sciencefixion**
Just a quick fix to update all the variables in team_test so that all tests pass again.

### CODE REVIEW

**Review by gabichuelas — APPROVED** [review]
Good catch @sciencefixion - Thanks!

File: test/team_test.rb [hunk 1]
```diff
@@ -5,13 +5,13 @@
 
 class TeamTest < Minitest::Test
   def setup
-    team_params = {
+    team_params = ({
```
Comment (rrabinovitch) [inline test/team_test.rb:8]: I'm curious about the addition of the parens here! My understanding was that adding them around a hash was only necessary if your test assertion expected value is a hash?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 coverage
+.DS_Store
```
Comment (rrabinovitch) [inline .gitignore:2]: ooh what's this?

File: test/team_test.rb [hunk 3]
```diff
@@ -29,9 +29,9 @@ def test_it_has_attributes
 
   def test_it_has_info_hash
     info_hash = @team.info
-    assert_instance_of Hash, hash
-    assert_equal 1, hash[:team_id]
-    assert_equal "ATL", hash[:abbreviation]
+    assert_instance_of Hash, info_hash
+    assert_equal 1, info_hash[:team_id]
+    assert_equal "ATL", info_hash[:abbreviation]
```
Comment (rrabinovitch) [inline test/team_test.rb:34]: ahh I see what the issue was now. good catch Dorion!

File: test/team_test.rb (hunk 1 above)
Comment (gabichuelas) [reply test/team_test.rb:8]: @rrabinovitch I think you're right, but it doesn't break the code so 🤷 ?

File: .gitignore (hunk 2 above)
Comment (gabichuelas) [reply .gitignore:2]: @rrabinovitch it's the thing that shows up when you mess around with your files via Finder GUI and not the terminal. `.gitignore` is where you want those. Or else they drive you insane.

Comment (rrabinovitch) [reply .gitignore:2]: word, thanks for explaining!

**Pull request merged**